Makefile

```diff
@@ -25,6 +26,12 @@ export COVERAGE_PATH
 export COVERAGE_PROFILE
 $(shell mkdir -p ${COVERAGE_PATH})
 
+GOPATH ?= $(shell go env GOPATH)
+GOMD2MAN ?= $(GOPATH)/src/cpuguy83/go-md2man
```

Comment:
Tha path is missing `github.com`. I'd love to first check if `go-md2man` is already installed on the system. The Skopeo Makefile is a good example how to do that. A `go install github.com/cpuguy83/...` should be enough to install the binary in $GOBIN.